io.github.Fndroid.clash_for_windows.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.Fndroid.clash_for_windows</id>
+  <launchable type="desktop-id">io.github.Fndroid.clash_for_windows.desktop</launchable>
+  <name>Clash for Windows</name>
+  <developer_name>Fndroid</developer_name>
+  <summary>A proxy tool.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://github.com/Fndroid/clash_for_windows_pkg/issues/153</project_license>
+  <url type="homepage">https://github.com/Fndroid/clash_for_windows_pkg</url>
+  <description>
+    <p>A Windows/macOS/Linux GUI based on Clash and Electron.</p>
```

Comment:
What does this actually do? Imagine you are explaining it to someone who is a smart computer user but has limited understanding about what Clash is, and explicitly doesn't care that this is a Windows/macOS application as well (this file will only every be seen or used on Linux).

Reply:
Thanks a lot, I've refined the description of the application.